sympy/functions/special/beta_functions.py

```diff
@@ -1,14 +1,33 @@
 from __future__ import print_function, division
-
-from sympy.core import S
 from sympy.core.function import Function, ArgumentIndexError
 from sympy.functions.special.gamma_functions import gamma, digamma
+from sympy.functions.special.hyper import hyper
+from sympy.functions.elementary.exponential import log
+from sympy import S, Dummy
+from sympy.core.logic import fuzzy_and
+
+class betabase(Function):
+    """
+    Abstract base class for beta functions.
+
+    This class is meant to reduce code duplication.
+
+    """
+    unbranched = True
+    def _eval_conjugate(self):
+        if len(self.args) > 2 :
+            a, b, z = self.args
+            return self.func(a.conjugate(), b.conjugate(), self.args[2].conjugate())
```

Comment:
I said to use properties for a, b and z so that it would not be necessary to check `len(args)`.